docs/configuration/extract-codacy-logs.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+#### Helper functions ####
+
+usage()
+{
+    echo "Usage: $0 -n <NAMESPACE> [-d <DAYS_OF_LOGS>]"
+    echo "When using the -d flag, <DAYS_OF_LOGS> must be between 1 and 7"
+    echo "Example: '$0 -n codacy -d 3' would retrieve the last 3 days of logs from the 'codacy' namespace of the kubernetes cluster"
+    exit 3
+}
+
+cleanup()
+{
+    if [ -d "$LOGS_DIR" ]; then
+        rm -r $LOGS_DIR &>/dev/null
+    fi
+}
+trap cleanup EXIT
+
+#### Log extraction script ####
+
+while getopts "n:d:" option; do
+    case ${option} in
+        n )
+            NAMESPACE=${OPTARG}
+            ;;
+        d )
+            if [ $OPTARG -le 0 ] || [ $OPTARG -ge 8 ]; then
+                usage
+            else
+                DAYS=$((${OPTARG}-1)) # Day 0 is today
+            fi
+            ;;
+        \? )
+            usage
+            ;;
+    esac
+done
+
+[ -z "$NAMESPACE" ] && usage
+
+CURRENT_DATE_TIME=$(date "+%Y%m%d-%H%M%S")
+LOGS_DIR="$(mktemp -d)/codacy_logs_$CURRENT_DATE_TIME"
+
+echo "Starting log files extraction"
+
+# Check if kubectl is available
+echo "Checking if kubectl is installed..."
+if ! kubectl version &>/dev/null; then
```

Comment:
Since @paulopontesm brought this we should have a variable called `KUBECTL` and then check both `kubectl`  and `microk8s.kubectl` and assign it.

Then we can use the variable instead of the command.

You can check for the binaries with `which`.